src/css/styles.css

```diff
@@ -151,15 +151,16 @@
   left: -60%;
   margin: 0 auto;
   text-align: center;
-  min-width: 100px;
   padding: 5px;
   background-color: var(--tooltip-bg);
   box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.2);
   border-radius: 4px;
   z-index: 99;
   display: block;
   transition: all 0.4s;
-  transform: translateY(calc(-100% - 15px));
+  transform: translateY(
+    calc(-100% - 0.6875rem)
```

Comment:
this break the centered position :upside_down_face:

Reply:
can you revert the changes of the slider tooltip ?

Reply:
It should be always aligned to the center.

Reply:
Smart tooltip position #94

Reply:
This line doesnt affect the sliders position, only the vertical alignment of the non slider tooltips. The tooltip was off centre due to a left margin style being applied that was causing the tooltip to appear off centre (as we now have less empty space on either side of the tooltip). I've pushed a change that will fix this.

Reply:
Should the non slider tooltip also be centered or left aligned?

Reply:
right aligned is fine